core/src/main/scala/caliban/schema/Schema.scala

```diff
@@ -239,18 +246,18 @@ trait GenericSchema[R] extends SchemaDerivation[R] with TemporalSchema {
       directives
     )
 
-  implicit val unitSchema: Schema[Any, Unit]             = scalarSchema("Unit", None, _ => ObjectValue(Nil))
-  implicit val booleanSchema: Schema[Any, Boolean]       = scalarSchema("Boolean", None, BooleanValue.apply)
-  implicit val stringSchema: Schema[Any, String]         = scalarSchema("String", None, StringValue.apply)
-  implicit val uuidSchema: Schema[Any, UUID]             = scalarSchema("ID", None, uuid => StringValue(uuid.toString))
-  implicit val shortSchema: Schema[Any, Short]           = scalarSchema("Short", None, IntValue(_))
-  implicit val intSchema: Schema[Any, Int]               = scalarSchema("Int", None, IntValue(_))
-  implicit val longSchema: Schema[Any, Long]             = scalarSchema("Long", None, IntValue(_))
-  implicit val bigIntSchema: Schema[Any, BigInt]         = scalarSchema("BigInt", None, IntValue(_))
-  implicit val doubleSchema: Schema[Any, Double]         = scalarSchema("Float", None, FloatValue(_))
-  implicit val floatSchema: Schema[Any, Float]           = scalarSchema("Float", None, FloatValue(_))
-  implicit val bigDecimalSchema: Schema[Any, BigDecimal] = scalarSchema("BigDecimal", None, FloatValue(_))
-  implicit val uploadSchema: Schema[Any, Upload]         = scalarSchema("Upload", None, _ => StringValue("<upload>"))
+  implicit val unitSchema: Schema[Any, Unit]             = scalarSchema("Unit", None, None, _ => ObjectValue(Nil))
+  implicit val booleanSchema: Schema[Any, Boolean]       = scalarSchema("Boolean", None, None, BooleanValue.apply)
+  implicit val stringSchema: Schema[Any, String]         = scalarSchema("String", None, None, StringValue.apply)
+  implicit val uuidSchema: Schema[Any, UUID]             = scalarSchema("ID", None, None, uuid => StringValue(uuid.toString))
+  implicit val shortSchema: Schema[Any, Short]           = scalarSchema("Short", None, None, IntValue(_))
+  implicit val intSchema: Schema[Any, Int]               = scalarSchema("Int", None, None, IntValue(_))
+  implicit val longSchema: Schema[Any, Long]             = scalarSchema("Long", None, None, IntValue(_))
+  implicit val bigIntSchema: Schema[Any, BigInt]         = scalarSchema("BigInt", None, None, IntValue(_))
+  implicit val doubleSchema: Schema[Any, Double]         = scalarSchema("Float", None, None, FloatValue(_))
+  implicit val floatSchema: Schema[Any, Float]           = scalarSchema("Float", None, None, FloatValue(_))
+  implicit val bigDecimalSchema: Schema[Any, BigDecimal] = scalarSchema("BigDecimal", None, None, FloatValue(_))
```

Comment:
I wonder if we can go ahead and provide urls for any of these. `UUID` for instance does have a specification I believe.

Reply:
You mean the official UUID spec, or something specific to GraphQL?

Reply:
Ah, the spec links to `https://tools.ietf.org/html/rfc4122`

Reply:
But we use `ID` for that, which is a standard scalar, and they say that standard scalars shouldn't have that URL.

Reply:
But maybe for the types in `TemporalSchema` could link to the relevant RFCs?